tableReader.js

```diff
@@ -1,62 +1,50 @@
-"use strict";
 
-/********  Variablen **********/
-
-// ein Objekt "person" wird angelegt
-const person = {
-    firstName: "Anton",
-    lastName: "Mustermann"
-}
-// Variable persons wird als Array angelegt
-let persons = [];
+// tableReader.js - Letze Änderung: Maurice König
 
+"use strict";
 
```

Comment:
```suggestion
/******** Variablen **********/
```
Ich würde gerne von der Art des Aufbaus und auch der Kommentierung nahe an den im Unterricht vorgestellten Dateien bleiben, also durchaus Abschnittsbezeichnungen und Kommentare zu Funktionen drin lassen, als Hilfe für die Einsteiger in die Programmierung.

Reply:
Pushe dann gleich nochmal die abgeänderte Version @HolgerSin!

Reply:
sorry wegen doppelter Kommentare hier, bin noch am rumprobieren wie das hier läuft mit reviews etc.

Reply:
Alles gut, ich habe gerade auch einen sonderbaren Pull gestellt. Muss mich mit den Mechaniken auch noch anfreunden.